sap/aibus/dar/client/model_manager_client.py

```diff
@@ -160,12 +162,18 @@ def create_job(
             dataset_id,
             model_template_id,
         )
-
-        payload = {
-            "modelName": model_name,
-            "datasetId": dataset_id,
-            "modelTemplateId": model_template_id,
-        }
+        if business_blueprint_id:
```

Comment:
Can you add a check to make sure that not both `model_template_id` and `business_blueprint_id` are given at the same time? I believe it is not supported in the backend, right? And maybe a test for that scenario as well?

Reply:
Yes added those additional checks and tests. Yes its not supported in backend as well.
Thanks !!